Automattic/Simplecons
# Update Publish Icon

### Fix
In this PR, I am deleting the 'published' and 'unpublished' icons to be replaced with a single 'publish' icon which is more consistent with the other icons in the set.

The new "publish" icon uses an arrow next to the existing "note" icon.

<img width="435" alt="Screenshot 2021-02-16 at 15 24 06" src="https://user-images.githubusercontent.com/36532468/108083479-043f4900-706b-11eb-8f68-a1af21a08e3f.png">

### To Test
Verify the icon appears as intended on a 24px grid.
Verify the SVG code is consistent with other icons.
